### ISSUE TITLE
reported cpu usage is sometimes wrongly zero

### ISSUE THREAD

**Issue by Jaxartes** [Jaxartes/logrun#1]
Sometimes, on some systems, the CPU usage reported at the end by "logrun" is inaccurate; when I've seen it it's been zero.  I haven't been able to reproduce this on the machines I use for developing "logrun" yet.

My best guess as to what's going wrong is, the child process isn't being wait()ed for.  And my best guess as to why it appears on some machines and not others, is the number of CPU cores.

Of course, sometimes the CPU usage really is close to zero.  That's not what I mean.

[28 Oct 2017 -- I originally noted that 'logrun "openssl speed md5 &"' produces similar results.  But that seems to be a different bug, and the use of "&" within logrun is  not an important use case.]

**Comment by Jaxartes** [comment]
Testing supports the notion that the problem occurs with larger numbers of cores.  I've tried on 2-core systems and not seen it, while on a 4-core one (Raspberry Pi 3 B) it happens.

I've got a fix that tests out so far, but it'll have to wait at least a week until I get a chance to work on it more.  The fix has a downside, it makes logrun linger about 0.1 second after the program is done.  That's OK for the ways I use logrun, but may be a problem for other use cases I haven't thought of.
